server/pogo/cards.go

```diff
@@ -0,0 +1,26 @@
+package pogo
```

Comment:
This is just a handy enum.  We will have to talk about file location for this stuff though.  Not being allowed cyclical dependencies is a real pain in the ass.

Reply:
It's annoying that it's hard enforced --- but I will say that every time I've redesigned things to remove circular dependencies, I've liked the layout better than before.